<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindReplace</class>
 <widget class="QDialog" name="FindReplace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>475</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>475</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Find &amp; Replace Src Defines</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>464</width>
     <height>267</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="lblCategory">
             <property name="text">
              <string>Category:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QComboBox" name="cboCategory"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="lblReplaceThis">
             <property name="text">
              <string>Replace this:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cboReplaceThis"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="lblReplaceWith">
             <property name="text">
              <string>With:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="cboReplaceWith"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>357</width>
             <height>8</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>112</height>
            </size>
           </property>
           <property name="title">
            <string>Apply to slices</string>
           </property>
           <widget class="QRadioButton" name="rbtnApplyAll">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>23</y>
              <width>71</width>
              <height>31</height>
             </rect>
            </property>
            <property name="minimumSize">
             <size>
              <width>71</width>
              <height>31</height>
             </size>
            </property>
            <property name="text">
             <string>All</string>
            </property>
           </widget>
           <widget class="QWidget" name="">
            <property name="geometry">
             <rect>
              <x>11</x>
              <y>68</y>
              <width>321</width>
              <height>33</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QRadioButton" name="rbtnApplyRange">
               <property name="minimumSize">
                <size>
                 <width>71</width>
                 <height>31</height>
                </size>
               </property>
               <property name="text">
                <string>Range</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spbRangeLower">
               <property name="minimumSize">
                <size>
                 <width>51</width>
                 <height>21</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblRangeTo">
               <property name="minimumSize">
                <size>
                 <width>25</width>
                 <height>31</height>
                </size>
               </property>
               <property name="text">
                <string>to</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spbRangeUpper">
               <property name="minimumSize">
                <size>
                 <width>51</width>
                 <height>21</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>17</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblReplaceResult">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="pbReplace">
        <property name="text">
         <string>Replace</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbClose">
        <property name="text">
         <string>Close</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>88</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
